akka-http/src/main/scala/akka/http/scaladsl/client/RequestBuilding.scala

```diff
@@ -18,37 +18,39 @@ import HttpMethods._
 trait RequestBuilding extends TransformerPipelineSupport {
   type RequestTransformer = HttpRequest ⇒ HttpRequest
 
+  implicit val timeout = Timeout(1.second)
```

Comment:
Sure, but how would you manage the default timeout then (as currently)? Method signature defaults would result in conflicting overloads.